scripts/us_census/pep/annual_population/README.md

```diff
@@ -0,0 +1,47 @@
+# US Census PEP: National Population Count
+
+## About the Dataset
+This dataset has Annual Population Estimates from the year 1900 to 2020 for different geographic level such as National, State, County, and Cities.
+
+The population is categorized by [Count_Person](https://datacommons.org/browser/Count_Person) StatVar in Datacommons.org
+
+### Download URL
+A backend API allows the data to be queryable and responds with txt/xls/xlsx data. The API endpoint is at "https://www2.census.gov/programs-surveys/popest/tables/1900-1980/national/totals/popclockest.txt","https://www2.census.gov/programs-surveys/popest/tables/1980-1990/state/asrh/st0009ts.txt","https://www2.census.gov/programs-surveys/popest/tables/1900-1980/counties/totals/e7079co.txt","https://www2.census.gov/programs-surveys/popest/datasets/2010-2020/counties/totals/co-est2020.csv". The API takes URL as a string input and downloads the dataset.
```

Comment:
Please seem my comment about README in https://github.com/datacommonsorg/data/pull/658